src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -4139,7 +4140,7 @@ module TcDeclarations =
     //-------------------------------------------------------------------------
 
     /// Bind a collection of mutually recursive definitions in an implementation file
-    let TcMutRecDefinitions (cenv: cenv) envInitial parent typeNames tpenv m scopem mutRecNSInfo (mutRecDefns: MutRecDefnsInitialData) =
+    let TcMutRecDefinitions (cenv: cenv) envInitial parent typeNames tpenv m scopem mutRecNSInfo (mutRecDefns: MutRecDefnsInitialData) skipExternalAugmWarning =
```

Comment:
I'd rather `skipExternalAugmWarning` was called `isMutRec`.  I believe that's accurate and the flag may have other similar uses later.